Automattic/wp-calypso
# Create a separate signup flow for paid media

#### Proposed Changes

* Creates a separate onboarding flow to be used with paid media landing pages.
* This flow will be used only for WPcom paid media, so I removed a lot of the conditional logic that's used by other A8C properties.
* This v1 is essentially a duplicate of the current experience. We'll optimize this for paid media in future PRs.
* I removed most of Lodash and other external libraries.
* More context available at pcbrnV-4Kk-p2/

#### Details
- Flow name: `onboarding-pm`
- Steps: [ `user-pm`, `domains-pm`, `plans-pm` ]

#### Testing Instructions

- Checkout this PR and start Calypso locally
- Navigate to: `/start/onboarding-pm`
- Proceed through the flow in both mobile and desktop viewports
- Verify that you can create both Free and Paid sites
- Test the `Use your own domain` flow.
- Test selecting a variety of domain categories (custom, featured, free)
- Inspect the Tracks events and confirm that they are reporting the correct Flow and Step names

#### Pre-merge Checklist





Related to #
